cmd/embedded-cluster/support_bundle.go

```diff
@@ -45,10 +48,10 @@ func supportBundleCommand() *cli.Command {
 					Writer:       stdout,
 					ErrWriter:    stderr,
 					LogOnSuccess: true,
+					Env:          map[string]string{"KUBECONFIG": kubeConfig},
```

Comment:
What's the reason for changing this from the `--kubeconfig` flag to passing it as an env var? Its likely that both are valid options, but I'm unclear on the reason we are changing this.

Reply:
It seems like `--kubeconfig` flag denotes intention so it fails if the file does not exist.

```
root@divine-sea:/home/developer# ./support-bundle --kubeconfig /does/not/exist ./spec.yaml
Error: failed to convert kube flags to rest config: stat /does/not/exist: no such file or directory
root@divine-sea:/home/developer# KUBECONFIG=/does/not/exist ./support-bundle ./spec.yaml


A support bundle was generated and saved at support-bundle-2024-10-24T13_29_49.tar.gz. Please send this file to your software vendor for support.
root@divine-sea:/home/developer#
```